cmd/fhttp/main.py

```diff
@@ -1,6 +1,6 @@
 import argparse
 import logging
-from func_python.http import serve
+from func_http.http import serve
```

Comment:
looking _now_ at this I see that `func_http.http` is a bit unfortunate as well :sweat_smile: